platform/viewer/src/components/ViewportGrid.tsx

```diff
@@ -190,46 +216,55 @@ function ViewerViewportGrid(props) {
           );
         }
 
+        // if we already have the displaySet in one or several of the viewports
+        // then figure out which one to navigate to
+        const matchingViewports = viewports.filter(viewport =>
+          viewport.displaySetInstanceUIDs.includes(
+            referencedDisplaySetInstanceUID
+          )
+        );
+        if (matchingViewports.length) {
+          const viewport = findMeasurementViewport(
+            matchingViewports,
+            viewportIndex,
+            measurement
+          );
+          // TODO - check the orientation of the viewport, and
+          // update it to match the provided image, changing the
+          // camera and type as required.
+          commandsManager.runCommand('jumpToImage', { viewport, imageIndex });
```

Comment:
having `platform/viewer` logic calling a command that will run inside extensions is very werid, and not ideal at all. We should avoid these calls, as you can see there is not other instance of `runCommand` inside platform/viewer folder